scenes/quests/story_quests/shjourney/2_shjourney_intro/template_intro_components/EffectSounds/_demon-laugh.wav.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: OleSouWester
+SPDX-License-Identifier: LicenseRef-CC-BY-4.0
+
+DEMON LAUGH.wav by OleSouWester -- https://freesound.org/s/692035/ -- License: Creative Commons 0
```

Comment:
You've got contradictory information here and in every subsequent file.. Is this file under CC-BY-4.0 or CC0?

It appears the answer is CC0, in which case you should adjust the `SPDX-License-Identifier` line:


```suggestion
SPDX-License-Identifier: LicenseRef-CC0-1.0

DEMON LAUGH.wav by OleSouWester -- https://freesound.org/s/692035/ -- License: Creative Commons 0
```